packages/web-components/package.json

```diff
@@ -18,6 +19,12 @@
   "main": "dist/esm/index.js",
   "types": "dist/web-components.d.ts",
   "unpkg": "dist/web-components.min.js",
+  "exports": {
+    "./text/define": {
```

Comment:
What is the use case for using this import vs package import?
I guess I am supposed to import `components/define` for all used components to register the custom elements and package import anything else?
Perhaps worth adding a `readme` with a brief Usage section.

Reply:
This allows one-off imports of individual components as each will have a unique path in the package.

```ts
import "@fluentui/web-components/text";
```

Reply:
Following up on this, the above example doesn't represent what was originally in the PR, so revisiting this...I think one reason that we went with `./text/define` which would be used as below was to leave room for the opportunity for a text specific path to export _more_ than just the defined element. I think that is probably too complex and everything is available via the root of the package, so I've updated this to be available just as "text" per above because it's more intuitive than a "define" path IMO. We can adjust if need be, but this makes more sense than what was originally there (shown below).
```ts
import "@fluentui/web-components/text/define";
```